content/source/docs/enterprise/migrate/workspaces.html.md

```diff
@@ -59,94 +55,86 @@ In your shell, go to the directory with the Terraform configuration you're migra
-
-## Step 9: Migrate `default` by Running `terraform init`
-
-Run `terraform init` to migrate the default workspace.
-
-Terraform will ask two questions:
-
-- **"Do you want to copy only your current workspace?"** Answer "yes."
+- Use the `remote` backend.
+- In the `organization` attribute, specify the name of your TFE organization.
+- The `hostname` attribute is only necessary with private TFE instances. You can omit it if you're using the SaaS version of TFE.
+- In the `prefix` attribute in the `workspaces` block, specify a prefix to use when naming newly created TFE workspaces. You'll use short workspace names on the CLI, and long workspace names with the prefix added in TFE's UI and API. See [the remote backend documentation][remote-backend] for details.
 
-    Terraform init can only handle the default workspace, so you'll use a different command for the others.
-- **"Do you want to copy existing state to the new backend?"** Answer "yes."
+## Step 6: Run `terraform init`, Answer "Yes", and Name the Default Workspace
 
-The `default` workspace is now migrated.
+Run `terraform init`.
 
-You can test the migration by unlocking the corresponding TFE workspace and queueing a plan; if the plan results in no changes or very small changes, the migration probably worked correctly.
+The init command will offer to migrate each previous state to a new TFE workspace. The prompt usually looks like this:
 
-## Step 10: Edit the Backend to Change Workspaces
+```
+Initializing the backend...
+Do you want to migrate all workspaces to "remote"?
+  Both the existing "local" backend and the newly configured "remote" backend
+  support workspaces. When migrating between backends, Terraform will copy
+  all workspaces (with the same names). THIS WILL OVERWRITE any conflicting
+  states in the destination.
 
-Edit your backend configuration, and change the `name` to the next TFE workspace you want to migrate. For example, if the next local workspace is `dev1`, its new name in TFE might be something like `example_corp/database-dev1`.
+  Terraform initialization doesn't currently migrate only select workspaces.
+  If you want to migrate a select number of workspaces, you must manually
+  pull and push those states.
 
-At this point, it's a good idea to use a checklist of workspaces to keep track of your progress.
+  If you answer "yes", Terraform will migrate all states. If you answer
+  "no", Terraform will abort.
 
-## Step 11: Run `terraform init` and Answer "No"
+  Enter a value:
+```
 
-Run `terraform init`.
+Answer "yes."
 
-When asked if you want to migrate state, **answer "no."**
+Some backends, including the default `local` backend, allow a special `default` workspace that doesn't have a specific name. If you previously used a combination of named workspaces and the special `default` workspace, the prompt will next ask you to choose a new name for the `default` workspace, since TFE doesn't support unnamed workspaces:
```

Comment:
Do we have any conventions/suggestions/best practices for the alternate `default` namespace? (This is more about my own curiosity and not a request to make a change).

Reply:
TBH, I don't think we know enough about how people are using alternate workspaces in CLI terraform. Part of why the default case was blowing up on migration for so long is that no one here thought it made any sense to keep using `default` once you had any named workspaces. 

If I think in terms of the order things would happen in... probably you'd start a new configuration on default, tweaking and adjusting some new infrastructure you're not using yet. Then you'd get it the way you liked it and start using it as production. Then you'd want to spin up an alternate instance of it to test something dangerous, so you'd make a new workspace. In that scenario, `default` would be prod. But I don't know about other scenarios. I have to just assume that if you're using alternate workspaces, you already had to think it through enough that you probably know what your default workspace is used for.